scipy-1.0/paper.tex

```diff
@@ -691,23 +691,22 @@ \subsection*{Language choices}
 Fortran, despite its age, is still a high-performance scientific programming language with
 continued contemporary usage\cite{Koelbel:1993:HPF:562354}. Thus, we wrap the following excellent, field-tested Fortran
 libraries in order to provide Python convenience while benefiting from their performance:
-FFTPACK\cite{SWARZTRAUBER198445, SWARZTRAUBER198251}, % for performing Fourier transforms,
-ODEPACK\cite{hindmarsh1983odepack}, % for solving ordinary differential equation initial value problems,
-QUADPACK\cite{piessens1983quadpack}, % for numerical integration of one-dimensional functions,
-FITPACK\cite{Dierckx:1993:CSF:151103}, % for curve-fitting and interpolation,
-ODRPACK\cite{ODRPACK_Boggs}, % for orthogonal distance regression,
-MINPACK\cite{osti_6997568}, % for minimization of linear and nonlinear equations,
```

Comment:
"[MINPACK](https://en.wikipedia.org/wiki/MINPACK) is a library of FORTRAN subroutines for the solving of systems of nonlinear equations, or the least squares minimization of the residual of a set of linear or nonlinear equations."

Not really general numerical nonlinear minimization, as this might suggest.